composer.json

```diff
@@ -21,7 +21,7 @@
     ],
     "require": {
         "php": "^8.0.2",
-        "craftcms/cms": "^4.0.0-alpha",
+        "craftcms/cms": "^4.3.5",
```

Comment:
PHPStan called out that the `onInit` method we're using in Module.php is only available in `>=4.3.5` 

Gotta love static analysis